packages/jellyfish-api-core/__tests__/category/blockchain/getChainTxStats.test.ts

```diff
@@ -0,0 +1,90 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { Testing } from '@defichain/jellyfish-testing'
+
+describe('ChainTxStats', () => {
+  const testing = Testing.create(new MasterNodeRegTestContainer())
+
+  beforeAll(async () => {
+    await testing.container.start()
+  })
+
+  afterAll(async () => {
+    await testing.container.stop()
+  })
+
+  it('should getChainTxStats for selected interval', async () => {
+    const blockHash = await testing.misc.waitForBlockHash(3)
+    const stats = await testing.rpc.blockchain.getChainTxStats(2, blockHash)
+
+    expect(stats.time).toBeGreaterThan(1550000000)
```

Comment:
personally i feel like testing it to be greater than a constant to be not very meaningful. how about we declare `const dtBefore = Date.now() / 1000` before we wait and we test against `dtBefore` ?

Reply:
I believe depending on Date.now() generally introduces non-determinism / flakiness, so it should be avoided where possible - based on other tests I see this pattern too. Wdyt?

Reply:
personally i do not see the non-determinism here as `waitForBlockHash` will definitely require some amount of time for the chain to generate 3 blocks

Reply:
the _some amount of time_ is the non-deterministic part I think. Noobie request 😄 - I wanna know what others think too so that I know what the general strategy is when it comes to checking time fields

Reply:
i think checking the tip block is more clarify
eg:
```ts
const bestHash = await testing.container.getBestBlockHash()
const bestBlock = await testing.container.call('getblock', [bestHash, 1])
expect(stats.time).toStrictEqual(bestBlock.mediantime)
```